### PR TITLE
Quotes on `/vs/github-codespaces` page

### LINKED ISSUE #3510: [vs/codespaces] add quotes
### Description

Implement internal task [[1](https://www.notion.so/gitpod/Add-quotes-to-vs-codespaces-527551ecf87842ee9d1c3f5248b31b86)]

### PR DESCRIPTION
**Pull request by Siddhant-K-code**
## Description

Quotes on `vs/codespaces` page

## Related Issue(s)

Fixes #3510

## How to test

- Open [`/vs/github-codespaces`](https://deploy-preview-3512--gitpod-kumquat.netlify.app/vs/github-codespaces) & see :eyes:

## Release Notes

```release-note
NONE
```

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3512"></a>

### CODE REVIEW

File: src/lib/contents/codespaces.ts [hunk 1]
```diff
@@ -10,6 +10,58 @@ import type {
...
+    name: "Casey LP",
+    avatar: "casey-lp.jpeg",
+    org: ``,
+    role: "DevOps Engineer",
+    text: `
+      <p>
+      After a year spent with Gitpod and Codespaces, my particular stand out choice is - Gitpod. I've found Gitpod to be more flexible, easier to start with, generally more stable, and comes with a vibrant and helpful community over on Discord.
+      </p>
+    `,
+  },
```
Comment (Siddhant-K-code) [inline src/lib/contents/codespaces.ts:45]: Point to be noted!

File: src/lib/contents/codespaces.ts [hunk 2]
```diff
@@ -10,6 +10,58 @@ import type {
...
+      `,
+  },
+  /**
+   * Avatar image taken from: https://github.com/soup-bowl
+   * Blog post: https://soupbowl.blog/2023/02/a-year-of-cloud-code-in-review
+   */
+  {
+    name: "Casey LP",
+    avatar: "casey-lp.jpeg",
+    org: ``,
```
Comment (MichaelAring) [inline src/lib/contents/codespaces.ts:38]:
```suggestion
    org: `
    <span>
     Soupbowl
     </span>
    `,
```

File: src/lib/contents/codespaces.ts [hunk 3]
```diff
@@ -10,6 +10,58 @@ import type {
...
+    role: "DevOps Engineer",
```
Comment (MichaelAring) [inline src/lib/contents/codespaces.ts:39]:
```suggestion
    role: "DevOps Engineer and Author at",
```

**Review by MichaelAring — CHANGES_REQUESTED** [review]
Added blog of Casey LP

File: src/lib/contents/codespaces.ts (hunk 3 above)
Comment (Siddhant-K-code) [reply src/lib/contents/codespaces.ts:39]: It is their personal blog. So, How about `Author of Soupbowl.blog` ?
```suggestion
    role: "DevOps Engineer and Author of",
```

File: src/lib/contents/codespaces.ts (hunk 2 above)
Comment (Siddhant-K-code) [reply src/lib/contents/codespaces.ts:38]: We need to clarify it to `Soupbowl.blog`

Comment (MichaelAring) [reply src/lib/contents/codespaces.ts:38]: Agreed

**Pull request merged**